semantic/src/site/globals/site.overrides

```diff
@@ -155,23 +155,23 @@ p strong {
 }
 
 // "Introduction" subheader
-#application-area .ui.header .sub.header {
+.ui.header .sub.header {
   margin-top: 11px;
   margin-bottom: 16px;
   font-size: @fontLarge;
   letter-spacing: @letterSpacingSmall;
 }
 
 // Extra font and margin on Steps header on application home/create
-#application-area .steps-header {
+.steps-header {
   font-size: @fontSmall;
   font-weight: @regularWeight;
   margin-top: 0;
   margin-bottom: 5px;
 }
```

Comment:
Hmmm I tried to move to `header.overwrites` but I'm getting the same problem... : (

Reply:
That's odd -- do you have an recent commit I can have a quick look at? I'd like to properly understand what's going on here.

Reply:
I ended up checking that this header didn't need a lot of overwriting. The font size and weight using the ones for h4 are nice (probably why I didn't realise this wasn't working), just the margin-top and bottom needed to be smaller.

Please have a look on what was done here: ce73e9c

Reply:
Cool, that looks good. I don't fully understand why you need the `!important`, but it must be that `.ui.header` has a higher specificity than `.steps-header`. Getting a handle of the quirks of specificity precedence is quite confusing, so I think this is probably a good case for using `!important`. 

(I think it's because `.ui` is also applied to the parent element (`ui container`), which makes `ui.header` a more specific version of `.ui`, so it takes precedence over `steps-header`. But I need to understand precedence with overlapping classes better.)